etc/profile-a-l/ani-cli.profile

```diff
@@ -0,0 +1,72 @@
+# Firejail profile for ani-cli
+# Description: Shell script to watch Anime from the terminal
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include ani-cli.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${DOWNLOADS}
```

Comment:
Drop this line. ${DOWNLOADS} is never blacklisted by disable-xdg.inc (as is ${DESKTOP}.